www/include/reporting/dashboard/template/viewServicesLog.ihtml

```diff
@@ -118,6 +118,12 @@
 	</tr>
 </table>
 
+{literal}
+<script>
+	formatDateMoment();
+</script>
+{/literal}
+
```

Comment:
Maybe, Using script and literal tags in duplicate ?

Reply:
change done